blueoil/converter/templates/include/global.tpl.h

```diff
@@ -31,12 +31,23 @@ limitations under the License.
 #endif
 using QUANTIZED_PACKED_KERNEL = QuantizedPacked<{{ params.default_qword_dtype.cpptype() }}>;
 
+#ifdef AARCH32
 #define IP_CSR_ADDR 0xFF200000
-#define INPUT_ADDR 0x20000000
-#define OUTPUT_ADDR 0x28000000
-#define KERNEL_ADDR 0x38000000
-#define THRESHOLD_ADDR 0x3F000000
+#define HW_BASE_ADDR 0x20000000
+#else
+#define IP_CSR_ADDR 0xB0000000
+#define HW_BASE_ADDR 0x1F000000
```

Comment:
Where `0xB0000000 ` and `0x1F000000` values comes from?
General one?
It's better to add link to the documentation.

Reply:
Those address values are temporal value.
There is no default value about them because different board has different value.
Actually it's better to make them variable to fit various hardware but this time current implementation just can use two case (arm and aarch64).
Maybe we can implement more flexible structure, but it should be another PR.

Reply:
@tk26eng OK. I understood. Improve in another PR is reasonable